packages/central-server/src/apiV2/userAccounts/RegisterUserAccounts.js

```diff
@@ -74,36 +66,7 @@ export class RegisterUserAccounts extends CreateUserAccounts {
   }
 
   async createRecord() {
-    const {
-      firstName,
-      lastName,
-      emailAddress,
-      employer,
-      position,
-      contactNumber,
-      password,
-    } = this.newRecordData;
-
-    let userData = {
-      firstName,
-      lastName,
-      emailAddress,
-      employer,
-      position,
-      contactNumber,
-      password,
-    };
-
-    // Get one of the non-default platform configs if it exists
-    const email = this.req?.apiClientUser?.email;
-    const platformConfig = email in PLATFORM_CONFIGS ? PLATFORM_CONFIGS[email] : null;
-
-    if (platformConfig) {
-      const { permissionGroupName, countryName, primaryPlatform } = platformConfig;
-      userData = { ...userData, permissionGroupName, countryName, primaryPlatform };
-    }
-
-    const { userId } = await this.createUserRecord(userData);
```

Comment:
There's currently a permissions check in this class that allows users to be created by anyone. Will adding your suggestion override this?

```  
async assertUserHasAccess() {
    await this.assertPermissions(allowNoPermissions); // new registrations can be created by anyone
  }
```

Reply:
I think it should still work correctly. It would mean that:
- Anyone can register a user, but they can only give them permissions to a country/permission group that they have access to
- Anonymous registration should work okay since the api-client user has public permissions to the appropriate countries (and in the case of lesmis, the api-client has 'LESMIS Public' access to Laos)

Definitely worth testing this though to double check.

Reply:
I might be misunderstanding what you mean, so please correct me if I am wrong!
I think this particular endpoint here is just for when someone is registering through the form on Tupaia etc. There is a different handler for when someone is registering users via the admin panel, which currently has some permissions checks but only for admin panel access. Is this change you suggested intended mainly for the admin panel use case?